modules/p2/ting_carousel/css/ting-carousel-compact.css

```diff
@@ -52,7 +52,7 @@ filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#a6000000', e
   font-size: 0.9em;
   line-height: 1.25em;
   text-align: left;
-  margin: 5px 0;
+  margin-top: 5px;
```

Comment:
Is it possible to override this in `_carousel.css` instead of manipulating this file?